ibc-testkit/src/testapp/ibc/core/types.rs

```diff
@@ -139,7 +139,7 @@ pub struct MockClientConfig {
     trusting_period: Duration,
     #[builder(default = Duration::from_millis(3000))]
     max_clock_drift: Duration,
-    #[builder(default = Duration::from_secs(128000))]
+    #[builder(default = Duration::from_secs(128_000))]
```

Comment:
I liked the tidy outcome from clippy, however, couldn’t catch them on my machine. What version of clippy do you use? Did the CI detect these cases?

As for the particular case of the `128_000`, it's strange that 64000 would be considered acceptable.

Reply:
I cherry picked the lints after running all clippy lint groups. We shouldn't do this directly on CI (but probably locally for reviewing PRs is fine) - as most of them are very opinionated. But we can definitely add a bunch of them on CI.

Reply:
For the particular case, the lint case is called [`unreadable_literal`](https://rust-lang.github.io/rust-clippy/master/index.html#/unreadable_literal). Looks like they warn this when the decimal has 6 digits.

Nonetheless, I am ok with making `64_000` :+1:

Reply:
Let's fix things when the linter catches them. That's alright.